dms/models/directory.py

```diff
@@ -89,11 +93,8 @@ def _default_parent_id(self):
             return False
 
     group_ids = fields.Many2many(
-        comodel_name="dms.access.group",
```

Comment:
Why removing the explicit relation table and fields? Is it better to fix it to avoid changes in the future if the ORM changes.